### PR TITLE
GSdx: Adjust DBZ BT2 CRC hacks and id, add R&C3 id, change skipdraw GUI limit

### PR DESCRIPTION
**Pull request by lightningterror**
Adjust region id for BT2.
Move the sky texture(depth) hack back to Partial level only for the EU regions.  Effect is still not rendered correctly and causes a half screen bottom issue (#2022)(#1339).

GSdx: Change Skipdraw Maximum limit value.
Previously the limit was 1000, now 10000 in the GUI. It should help in
some rare cases where a higher number is needed without the need of ini
editing and value reset issues caused by the GUI.

### CODE REVIEW

**Comment by lightningterror** [comment]
I just realized that I created this PR master fork ... oh well.

**Comment by FlatOutPS2** [comment]
If you based the GitS changes solely on the gsdump, did you ask someone to confirm it does not cause any issues elsewhere in the game?

**Comment by lightningterror** [comment]
I actually have the game so I did my testing playing but not much.
D3D should be the same since the code wasn't changed for it.
OpenGL has the hack adjusted so that can be further tested.

@ivan89el wanna do a bit more testing?
You can download the build here
https://ci.appveyor.com/project/gregory38/pcsx2/build/1.3702-master/job/mdrxexya1bdfu8s3/artifacts

**Comment by FlatOutPS2** [comment]
Then why ask for a gsdump? :S

You should also be clearer that you change the skipdraw limit with this PR. Even looking at the code I might've missed that one. Is the hyper extreme skipdraw setting still necessary with the updated CRC hack?

**Comment by lightningterror** [comment]
Was lazy to search for the disc :stuck_out_tongue: 
Yeah the high skipdraw is needed because it uses a channel shuffle effect which is not implemented properly yet.

**Comment by turtleli** [comment]
I think you should also make the same change to the skipdraw range on the Linux GUI as well. I'd also prefer if you split the GUI change into a separate commit.

**Comment by lightningterror** [comment]
@bositman did some more testing and everything is good. Aside from shadows not working with the gamefix patch but the patch is needed to go through a door in the beginning. After that gamefixes can be disabled.

**Comment by turtleli** [comment]
Did you comment on the wrong PR?

**Comment by lightningterror** [comment]
No, he was testing GITS Stand Alone Complex so I thought I'd share his test results here as well, oh seems that I forgot to mention which game earlier.

**Comment by FlatOutPS2** [comment]
It kind of looks like you were telling bositman you did the testing. 😛

**Comment by bositman** [comment]
He told me to test it and so I did :P

**Comment by lightningterror** [comment]
> It kind of looks like you were telling bositman you did the testing.

It kinda does, but if you look closely I never used **I** in the sentence :)

**Comment by lightningterror** [comment]
I decided to remove the GITS update, I want to do some revisiting on that but in the future.

**Comment by FlatOutPS2** [comment]
Looks good to merge then. I might have to rebase my skipdraw offset PR because of 10000 change.

File: plugins/GSdx/GSHwHack.cpp [hunk 1]
```diff
@@ -58,11 +58,17 @@ bool GSC_DBZBT2(const GSFrameInfo& fi, int& skip)
 	{
 		if(fi.TME && /*fi.FBP == 0x00000 && fi.FPSM == PSM_PSMCT16 &&*/ (fi.TBP0 == 0x01c00 || fi.TBP0 == 0x02000) && fi.TPSM == PSM_PSMZ16)
 		{
-			if (Dx_only) // Feel like texture shuffle but not sure
-				skip = 26; //27
+			// Alpha channel (red line issue has been fixed).
+			// Sky texture (Depth) is properly rendered on OpenGL only for the NTSC version. The PAL version still has some issues (half screen bottom issue).
+			if (g_crc_region == CRC::EU || Dx_only)
+			{
+				skip = 26; // 27
```
Comment (turtleli) [inline plugins/GSdx/GSHwHack.cpp]: Maybe remove the `// 27`?

Comment (turtleli) [reply plugins/GSdx/GSHwHack.cpp]: Test both, see what's better, and keep only that value then. Comment was introduced in 357030330a587aa51e1461a0b8009919b40be594.

Comment (FlatOutPS2) [reply plugins/GSdx/GSHwHack.cpp]: Based on the commit message the 27 value was used when it only worked on the PAL version. Either it was just a general correction, or using the higher value removed something it shouldn't on the NTSC versions.

Comment (lightningterror) [reply plugins/GSdx/GSHwHack.cpp]: > or using the higher value removed something it shouldn't on the NTSC versions

Yeah it was that. Feast your eyes on this, looks lovely doesn't it with 27 :smile: 

pcsx2 2018-02-05 15-30-10-92

**Comment by lightningterror** [comment]
You can merge it now.

**Pull request merged**